jobs/JOBSPROC_DUMP_MONITOR

```diff
@@ -0,0 +1,277 @@
+-${PACKAGEROOT:?}/obsproc.${obsproc_ver:?\
+"###FATAL ERROR \"obsproc_ver\" is not set"}} #doesn't matter if HOMEobsproc set
+
+export SCRIPTSobsproc=${SCRIPTSobsproc:-$HOMEobsproc/scripts}
+export EXECobsproc=${EXECobsproc:-$HOMEobsproc/exec}
+export USHobsproc=${USHobsproc:-$HOMEobsproc/ush}
+
+# ---------------------------------------
+# Paths for dump and bufr_dumplist items
+# ---------------------------------------
+
+export HOMEbufr_dump=${HOMEbufr_dump:\
+-${PACKAGEROOT:?}/bufr_dump.${bufr_dump_ver:?\
+"###FATAL ERROR \"bufr_dump_ver\" is not set"}}
+
+export USHbufr_dump=${USHbufr_dump:-$HOMEbufr_dump/ush}
+export FIXbufr_dump=${FIXbufr_dump:-$HOMEbufr_dump/fix}
+
+# NOTE: path to dump executables defaults to $HOMEbufr_util/exec in dumpjb
+
+# directory path to data dump script executed in model script
+# -----------------------------------------------------------
+export ushscript_dump=${ushscript_dump:-$USHbufr_dump}
+
+###############################################
+# Add some prod utilities to working directory
+###############################################
+echo "step ############# break ##############################" > ./break
+cp $UTILROOT/ush/err_chk   .; chmod +x err_chk
+cp $UTILROOT/ush/err_exit  .; chmod +x err_exit
+cp $UTILROOT/ush/postmsg   .; chmod +x postmsg
+cp $UTILROOT/ush/setpdy.sh .; chmod +x setpdy.sh
+
+##########################################
+# Run setpdy and initialize PDY variables
+##########################################
+set +x; echo -e "\n---> path to setpdy.sh below is: `which setpdy.sh`"; set -x
+setpdy.sh
+. $DATA/PDY
+
+#########################
+# Define COM directories
+#########################
+export COMIN_ROOT=${COMIN_ROOT:-${COMROOT:-""}}
+
+if [[ "$RUN_ENVIR" == nco ]]; then
+  export COMIN=${COMIN:-$(compath.py ${envir}/${obsNET}/${obsproc_ver}/${RUN}.${PDY})}
+  export COMOUT=${COMOUT:-$(compath.py ${envir}/${obsNET}/${obsproc_ver}/${RUN}.${PDY})}
+  mkdir -m 775 -p $COMOUT
+else
+  export COMIN=${COMIN:-$(compath.py ${envir}/${obsNET}/${obsproc_ver}/${RUN}.${PDY})}
+
+# COMOUT_ROOT for developers defaults to unique $DATA/com directory as root to
+#  prevent job from writing into a non-unique directory that might already have
+#  output in it from a previous checkout run for the same cycle
+# -----------------------------------------------------------------------------
+  export COMOUT_ROOT=${COMOUT_ROOT:-${DATA}/com}
+  export COMOUT=${COMOUT:-$(compath.py ${envir}/${obsNET}/${obsproc_ver}/${RUN}.${PDY})}
+  mkdir -m 755 -p $COMOUT
+fi
+
+# comin and comin_m1 are paths to dump status files for bufr_avgdata.sh
+export comin=${comin:?}/obsproc/${obsproc_ver}/${RUN}.${PDY}  #To be used in the future
+export comin_m1=${comin_m1:-${COMIN_ROOT:?}/obsproc/${obsproc_ver}/${RUN}.${PDYm1}}
+
+export comin=${comin:-$(compath.py ${envir}/${obsNET}/${obsproc_ver}/${RUN}.${PDY})}
+export comin_m1=${comin_m1:-$(compath.py ${envir}/${obsNET}/${obsproc_ver}/${RUN}.${PDY})}
```

Comment:
Careful -- comin_m1 is for PDYm1. Change to:
export comin_m1=${comin_m1:-$(compath.py ${envir}/${obsNET}/${obsproc_ver}/${RUN}.${PDYm1})}